tests/test_model.py

```diff
@@ -205,8 +206,10 @@ def test_full_forward_and_backward_gpt2_small(prefixlm, batch_size=2):
     else:
         neo_cfg.model.name = 'hf_causal_lm'
 
-    model = COMPOSER_MODEL_REGISTRY[neo_cfg.model.name](
-        neo_cfg.model, neo_cfg.tokenizer).to(device)
+    tokenizer = build_tokenizer(neo_cfg.tokenizer)
+
+    model = COMPOSER_MODEL_REGISTRY[neo_cfg.model.name](neo_cfg.model,
+                                                        tokenizer).to(device)
```

Comment:
to avoid the silent issues from before, can you add an assert here that `model.tokenizer` is the actually a tokenizer? (same for the other test)